packages/manager/src/features/Volumes/VolumeCreate.test.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import { VolumeCreate } from './VolumeCreate';
+
+const blockStorageEncryptionEnabledMock = vi.hoisted(() => {
+  return {
+    useIsBlockStorageEncryptionFeatureEnabled: vi.fn(),
+  };
+});
+
+describe('VolumeCreate', () => {
+  // @TODO BSE: Remove feature flagging/conditionality once BSE is fully rolled out
+  vi.mock('src/components/Encryption/utils.ts', async () => {
+    const actual = await vi.importActual<any>(
+      'src/components/Encryption/utils.ts'
+    );
+    return {
+      ...actual,
+      __esModule: true,
+      useIsBlockStorageEncryptionFeatureEnabled: blockStorageEncryptionEnabledMock.useIsBlockStorageEncryptionFeatureEnabled.mockImplementation(
+        () => {
+          return {
+            isBlockStorageEncryptionFeatureEnabled: false, // indicates the feature flag is off or account capability is absent
+          };
+        }
+      ),
+    };
+  });
+
+  it('should not have a "Volume Encryption" section visible if the feature flag is off and user does not have capability', () => {
+    const { queryByText } = renderWithTheme(<VolumeCreate />);
+
+    expect(queryByText('Encrypt Volume')).not.toBeInTheDocument();
+  });
+
+  it('should have a "Volume Encryption" section visible if feature flag is on and user has the capability', () => {
+    blockStorageEncryptionEnabledMock.useIsBlockStorageEncryptionFeatureEnabled.mockImplementationOnce(
+      () => {
+        return {
+          isBlockStorageEncryptionFeatureEnabled: true,
+        };
+      }
+    );
+
+    const { queryByText } = renderWithTheme(<VolumeCreate />);
+
+    expect(queryByText('Encrypt Volume')).toBeInTheDocument();
+  });
```

Comment:
There might be some value in using a more "e2e" approach so that `useIsBlockStorageEncryptionFeatureEnabled` is also tested instead of mocked. This approach does make it a bit harder to test the falsy case though

```suggestion
it('should have a "Volume Encryption" section visible if feature flag is on and user has the capability', async () => {
    server.use(
      http.get('*/v4/account', () => {
        return HttpResponse.json(
          accountFactory.build({ capabilities: ['Block Storage Encryption'] })
        );
      })
    );

    const { findByText } = renderWithTheme(<VolumeCreate />, {
      flags: { blockStorageEncryption: true },
    });

    await findByText('Encrypt Volume');
  })
```

Reply:
Opting to keep things as they are in the interest of moving things along for BSE, but we can have confidence in this pattern (was used for LDE also) and the general logic underlying `isBlockStorageEncryptionFeatureEnabled`